ansible/roles/common/tasks/main.yml

```diff
@@ -29,10 +29,25 @@
     block:
     - name: Reboot all VMs
       reboot:
-        connect_timeout: 300
         msg: "RHEL updates installing. Rebooting now."
-        pre_reboot_delay: 60
-        post_reboot_delay: 10
+        reboot_timeout: 60
+        connect_timeout: 5
+        pre_reboot_delay: 10
+        post_reboot_delay: 30
+      register: r_reboot
+    rescue:
+    - debug: var=r_reboot
+    - name: Recover from UNREACHABLE if any
+      ansible.builtin.meta: clear_host_errors
+    - name: Reboot the VMs again if the first attempt has failed
+      reboot:
+        msg: "RHEL updates installing. Rebooting now."
+        reboot_timeout: 60
+        connect_timeout: 5
+        pre_reboot_delay: 10
+        post_reboot_delay: 30
+      when: r_reboot is not succeeded
+    always:
```

Comment:
can you explain the motivation behind this change?
Does this happens a lot?
*All* provisions will go through this change, so we have to be careful when changing those values.

Reply:
During development, several times the reboot was failing because of a timeout or network disconnection (host Unreachable), causing all the deployment to fail. This block+rescue caused the provisioning not failing for this reason anymore, so I decided to maintain that peace of code